packages/universal-components/src/RangeDatePicker/components/ControlContainerItem.js

```diff
@@ -5,48 +5,82 @@ import { View } from 'react-native';
 import { defaultTokens } from '@kiwicom/orbit-design-tokens';
 
 import { Text } from '../../Text';
-import { StyleSheet } from '../../PlatformStyleSheet';
+import { StyleSheet, type StylePropType } from '../../PlatformStyleSheet';
+import { Touchable } from '../../Touchable';
 
 type Props = {|
   +label: string,
   +value: string,
+  +active?: boolean,
+  +style?: StylePropType,
+  +tabId: number,
+  +onPress: number => void,
 |};
 
 export default class ControlContainerItem extends React.Component<Props> {
+  static defaultProps = {
+    active: false,
+  };
+
+  onPress = () => {
+    this.props.onPress(this.props.tabId);
+  };
+
   render() {
-    const { label, value } = this.props;
+    const { label, value, active, style } = this.props;
     return (
-      <View style={styles.outerContainer}>
-        <View style={styles.arrow} />
-        <View style={styles.borderedContainer}>
-          <View style={styles.innerContainer}>
-            <View style={styles.dot} />
-            <View>
-              <Text style={styles.label}>{label}</Text>
-              <Text style={styles.date} weight="bold">
-                {value}
-              </Text>
+      <View style={styles.flexContainer}>
+        <Touchable onPress={this.onPress} style={styles.flexContainer}>
+          <View style={[styles.outerContainer, style]}>
+            {active && <View style={styles.arrow} />}
+            <View
+              style={[
+                styles.borderedContainer,
+                active && styles.borderedContainerActive,
+              ]}
+            >
+              <View style={styles.innerContainer}>
+                {active ? (
+                  <>
+                    <View style={styles.dot} />
+                    <View>
+                      <Text style={styles.label}>{label}</Text>
+                      <Text style={styles.date} weight="bold">
+                        {value}
+                      </Text>
+                    </View>
+                  </>
+                ) : (
+                  <Text style={styles.placeholder}>{label}</Text>
+                )}
+              </View>
             </View>
           </View>
-        </View>
+        </Touchable>
       </View>
     );
   }
 }
 
 const styles = StyleSheet.create({
+  flexContainer: {
+    flex: 1,
+  },
   outerContainer: {
     flex: 1,
   },
   borderedContainer: {
     borderWidth: parseFloat(defaultTokens.borderWidthInput),
-    borderColor: defaultTokens.borderColorButtonInfoBordered,
+    borderColor: defaultTokens.colorTextSecondary, // @TODO better name in defaultTokens
```

Comment:
What about using more universal `paletteInkLight` instead of text color value?

Reply:
Done